src/App.tsx

```diff
@@ -1,93 +1,97 @@
-        </form>
-      </header>
-
-      <section className="main">
-        <input
-          type="checkbox"
-          id="toggle-all"
-          className="toggle-all"
-          data-cy="toggleAll"
-        />
-        <label htmlFor="toggle-all">Mark all as complete</label>
+import { useTheme } from './hooks/use-theme';
 
-        <ul className="todo-list" data-cy="todoList">
-          <li>
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-view" />
-              <label htmlFor="toggle-view">asdfghj</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
+// import { TodoList } from './components/TodoList';
 
-          <li className="completed">
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-completed" />
-              <label htmlFor="toggle-completed">qwertyuio</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
+export const App: React.FC = () => {
+  const {
+    setUser,
+    userErrorHide,
+    setUserErrorHide,
+    userError,
+    changeUserButton,
+    setChangeUserButton,
+  } = useContext(Context);
+  const { theme, setTheme } = useTheme();
 
-          <li className="editing">
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-editing" />
-              <label htmlFor="toggle-editing">zxcvbnm</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
+  useEffect(() => {
+    const userOnLocaleStorage = localStorage.getItem('user');
 
-          <li>
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-view2" />
-              <label htmlFor="toggle-view2">1234567890</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-        </ul>
-      </section>
+    if (userOnLocaleStorage) {
+      setUser(JSON.parse(userOnLocaleStorage));
+    }
 
-      <footer className="footer">
-        <span className="todo-count" data-cy="todosCounter">
-          3 items left
-        </span>
+    setTheme('nigth');
+  }, []);
 
-        <ul className="filters">
-          <li>
-            <a href="#/" className="selected">All</a>
-          </li>
+  return (
+    <>
+      <Outlet />
 
-          <li>
-            <a href="#/active">Active</a>
-          </li>
+      <button
+        type="button"
+        className={classNames(
+          'theme-switcher',
+          theme === 'ligth' ? 'theme-switcher--ligth' : 'theme-switcher--nigth',
```

Comment:
```suggestion
          theme === 'ligth' ? 'theme-switcher--ligth' : 'theme-switcher--nigth',
          {
            ['theme-switcher--ligth']: theme === 'ligth',
            ['theme-switcher--nigth']: theme === 'nigth',
          }
```
as for me it's better, but event better to use map object for styles, where key will be theme (light or nigth)